hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/OzoneDelegationTokenSecretManager.java

```diff
@@ -436,6 +455,21 @@ private TokenInfo validateToken(OzoneTokenIdentifier identifier)
    */
   public boolean verifySignature(OzoneTokenIdentifier identifier,
       byte[] password) {
+    String secretKeyId = identifier.getSecretKeyId();
+    if (StringUtils.isNotEmpty(secretKeyId)) {
```

Comment:
The PR overall looks good. The only thing I have an opinion about is six months from now, I won't remember why we have an if else statement here. How about making this an intuitive helper method, like 
```suggestion
    if (isSymmetricKey(identifier)) {
```

or simply add a comment here.